content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -112,33 +112,36 @@ In fact, the Rich text editor offers a large bunch of possibilities!
 ![Records text area](../img/Records_text_area.png)
 
 # Display the products linked to a record (_since 3.2_)
-_Since the 3.2 version_, you can view directly from a reference entity record the first products linked to a record in a new `Products` tab.
+_Since the 3.2 version_, you can view the first products linked to a record in a new `Products` tab directly from a reference entity record.
```

Comment:
"you can see" instead of "you can view"?

Reply:
done